scripts/install.sh

```diff
@@ -14,6 +14,12 @@ if [[ ${#@} -lt 1 || ${#@} -gt 2 ]]; then
     exit 1
 fi
 
+openshift_install_check=$(kubectl get deployment olm-operator -n openshift-operator-lifecycle-manager -o=jsonpath='{.spec}' 2>/dev/null)
+if [[ $openshift_install_check != "" ]]; then
```

Comment:
You probably can just use the `z` flag to determine if this variable is empty (and/or set) but this looks fine to me.

I'll hold in case anyone else has any comments.

/lgtm
/hold